plugins/agents/muninpluginrunner/exec.go

```diff
@@ -0,0 +1,74 @@
+package muninpluginrunner
+
+import (
+	"bufio"
+	"regexp"
+	"strconv"
+
+	"github.com/abrander/agento/plugins"
+	"github.com/abrander/agento/timeseries"
+)
+
+func init() {
+	plugins.Register("muninpluginrunner", newMuninPluginRunner)
+}
+
+// MuninPluginRunner will retrieve stub status.
+type MuninPluginRunner struct {
+	Cmd string `toml:"cmd" json:"cmd" description:"Command to run"`
+	Arg string `toml:"arg" json:"arg" description:"Arguments to command"`
```

Comment:
Maybe we should name these `Command` and `Arguments` (Or `Argument` (singular))?

Reply:
@abrander fixed